### DISCUSSION TITLE
HTML Compare and find the difference in it like added removed element

### DISCUSSION THREAD

**Discussion by starsibi** [egil/AngleSharp.Diffing#45] [Q&A] [answered]
I want to compare two HTML files to identify any added or removed elements and display the differences. Will this package handle the operation? If so, is there a code example, or is there a wrapper package available for it?
Need like this for reference
![image](https://github.com/user-attachments/assets/215358b1-f00d-424c-9585-b5e5547bfd92)

**Comment by egil** [comment, marked as answer]
It does not include a visualizer itself, it just returns a collection of `IDiff` that represents the differences. You can use those to create whatever visualizations you like. See https://github.com/AngleSharp/AngleSharp.Diffing/wiki/Getting-Started
